test/unit/remote-config/condition-evaluator.spec.ts

```diff
@@ -960,10 +960,17 @@ describe('ConditionEvaluator', () => {
               targetCustomSignalValues: ['foo', 'bar']
             }
           });
+          const condition2 = createNamedCondition('is_active', {
```

Comment:
Nit: naming the different test cases something more self-descriptive than "is_enabled" and "is_active" would be helpful.